doc/development/development.rst

```diff
@@ -151,23 +151,70 @@ navigate to your local repository in your terminal and run the following command
 
     pre-commit install
 
-Now, each time you make a commit, pre-commit will automatically run black and any other configured hooks.
+Now, each time you make a commit, pre-commit will automatically run Black and any other configured hooks.
 If the hooks make changes or if there are any issues, the commit will be stopped, and you'll be able to review and add the changes.
 
-If you want black to omit a line from formatting, you can add the following comment to the end of the line:
+If you want Black to omit a line from formatting, you can add the following comment to the end of the line:
+
+.. code-block:: python
+
+    # fmt: skip
+
+To ignore a block of code you must flank the code with two comments:
 
 .. code-block:: python
 
     # fmt: off
+    code here
+    # fmt: on
 
 As described in the `black documentation <https://black.readthedocs.io/en/stable/the_black_code_style/current_style.html#code-style>`_,
 
-The following are some styling conventions that we follow in SpikeInterface:
+
+For docstrings, SpikeInterface generally follows `numpy docstring standard <https://numpydoc.readthedocs.io/en/latest/format.html#numpydoc-docstring-guide>`_.
+This includes providing a one line summary of a function, and the standard NumPy sections including :code:`Parameters`, :code:`Returns`, etc. The format used
+for providing parameters, however is a little different. The project prefers the format:
+
+.. code-block:: bash
+
+    parameter_name: type, default: default_value
+
+
+This allows users to quickly understand the type of data that should be input into a function as well as whether a default is supplied. A full example would be:
+
+.. code-block:: python
+
+    def a_function(param_a, param_b=5, param_c="mean"):
+        """
+        A function for analyzing data
+
+        Parameters
+        ----------
+        param_a: dict
+            A dictionary containing the data
+        param_b: int, default: 5
+            A scaling factor to be applied to the data
+        param_c: "mean" | "median", default: "mean"
+            What to calculate on the data
+
+        Returns
+        -------
+        great_data: dict
+            A dictionary of the processed data
+        """
+
+
+Note that in this example we demonstrate two other docstring conventions followed by SpikeInterface. First, that all string arguments should be presented
+with double quotes. This is the same stylistic convention followed by Black and enforced by the pre-commit for the repo. Second, when a parameter is a
+string with a limited number of values (e.g. :code:`mean` and :code:`median`), rather than give the type a value of :code:`str`, please list the possible strings
+so that the user knows what the options are.
+
```

Comment:
And we can have another small one here called "miscelleanous conventions" or something like this.

Reply:
Done : )

Reply:
Thanks!